spec/datetimeformat.html

```diff
@@ -282,6 +283,22 @@ <h1>BasicFormatMatcher ( _options_, _formats_ )</h1>
             1. If _format_ has a field [[&lt;_property_&gt;]], let _formatProp_ be _format_.[[&lt;_property_&gt;]]; else let _formatProp_ be *undefined*.
             1. If _optionsProp_ is *undefined* and _formatProp_ is not *undefined*, decrease _score_ by _additionPenalty_.
             1. Else if _optionsProp_ is not *undefined* and _formatProp_ is *undefined*, decrease _score_ by _removalPenalty_.
+            1. Else if _property_ is *"timeZoneName"*, then
+              1. If _optionsProp_ is *"short"* or *"shortGeneric"*, then
+                1. If _formatProp_ is *"shortOffset"*, decrease _score_ by _offsetPenalty_.
+                1. Else if _formatProp_ is *"longOffset"*, decrease _score_ by (_offsetPenalty_ + _shortMorePenalty_).
+                1. Else if _optionsProp_ is *"short"* and _formatProp_ is *"long"*, decrease _score_ by _shortMorePenalty_.
+                1. Else if _optionsProp_ is *"shortGeneric"* and _formatProp_ is *"longGeneric"*, decrease _score_ by _shortMorePenalty_.
+                1. Else if _optionsProp_ ≠ _formatProp_, decrease _score_ by _removalPenalty_.
```

Comment:
@FrankYFTang have you seen any other instances of this symbol being used? What about just writing down "is not equal to"?

Reply:
~/tc39/ecma402$ egrep -n "1..*≠" spec/*.html
spec/datetimeformat.html:285:            1. Else if _optionsProp_ ≠ _formatProp_, then
spec/numberformat.html:695:        1. If _f_ ≠ 0, then
spec/pluralrules.html:51:         1. If _f_ ≠ 0, then